packages/lmi/tests/test_cost_tracking.py

```diff
@@ -369,3 +369,673 @@ async def track_callback(response):  # noqa: RUF029
 
             # Callback should have been called exactly once
             assert len(callback_calls) == 1
+
+
+class TestCachedTokenCosts:
+    """Tests for cached token cost tracking and field extraction."""
+
+    @pytest.mark.asyncio
+    @pytest.mark.vcr
+    async def test_anthropic_cache_creation_cost_accuracy(self):
+        """
+        Audit that cache creation tokens are priced correctly.
+
+        Verifies that when cache_creation_input_tokens > 0, the cost calculation
+        applies the cache creation rate.
+
+        Tests Claude Sonnet 4.5 (latest Sonnet model with prompt caching).
```

Comment:
```suggestion
```

[style nit] don't duplicate test hypers such as model instance in the docstring, as people will change the `name=` in the future and forget to update here. It happens all the time